experimental/ExteriorAlgebra/ExteriorAlgebra_naive.jl

```diff
@@ -0,0 +1,95 @@
+export exterior_algebra_naive;  # constructor
+export ContainsDuplicates;   # export just for developing/testing
+
+# Auxiliary function -- worth keeping?
+
+function ContainsDuplicates(L)  # L must be indexable (1:n), and its elems must be sortable
+  n = length(L);
+  if (n < 2)  return false;  end;
+  if (n == 2)  return (L[1] == L[2]);  end;
+  sorted = sort(L);
+  for i=1:n-1  if (sorted[i] == sorted[i+1])  return true;  end;  end;
+  return false;
```

Comment:
Despite being correct, this is probably not the fastest way to do it. 
I would suggest something like 
```
for i in 1:n-1
  any(y->(y==L[i]), L[i+1:end]) && return true
end
return false
```

Reply:
Not sure.  Assuming `sort` is "soft-linear" then sorting and checking adjacent elements should be "soft-linear" (at the cost of making copies of the vector elements.  Your loop appears at first sight to be quadratic (but maybe Julia can compile it cleverly), so ought to be slower when the vector contains enough elements...

Reply:
Anyway, thanks for the suggestion.  I shall memorize the `any` function which will surely be useful at some point.

Reply:
try `any(isequal(L[i]), L[i+1:end])` don't know about speed but more julia...
`any(isequal(a), b)` is equivalent to `any(x->x==a, b)`

Reply:
In the end I used the built in Julia function `Base.allunique` which seems to do what I needed.